### PR TITLE
payment link added and blog url changed and some minor fixes

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/api/flavour.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+const getDevice = (details, regexp) => {
```
Comment (arrogant-ant) [inline src/api/flavour.js:2]: for consistency please rename this as getFlavour

File: src/components/organisms/o-payment-gateway.vue [hunk 2]
```diff
@@ -120,77 +96,58 @@ export default {
       BillText: "Bill description: ",
       bill: "Total : 2300 , discount : -300, GST : 500 , Total Amount: 2500 ",
```
Comment (arrogant-ant) [inline src/components/organisms/o-payment-gateway.vue:97]: are we over-riding bill or its static as of now

File: src/components/templates/t-discover.vue [hunk 3]
```diff
@@ -88,6 +86,15 @@ export default {
     searchedText: String,
     show: Boolean,
   },
+  computed: {
+    place() {
+      const temp = this.searchedText.indexOf("/");
```
Comment (arrogant-ant) [inline src/components/templates/t-discover.vue:91]: please add some sample searchText in comments

File: src/store/index.js [hunk 4]
```diff
@@ -1,13 +1,16 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import device from './modules/device'
+import blog from './modules/blog'
 
 Vue.use(Vuex)
 
 const store = new Vuex.Store({
     // !this is just for testing purpose
```
Comment (arrogant-ant) [inline src/store/index.js:9]: remove this comment, as this is not relevant now

Comment (alias01) [reply src/store/index.js:9]: done

File: src/components/templates/t-discover.vue (hunk 3 above)
Comment (alias01) [reply src/components/templates/t-discover.vue:91]: done

File: src/components/organisms/o-payment-gateway.vue (hunk 2 above)
Comment (alias01) [reply src/components/organisms/o-payment-gateway.vue]: made it dynamic now

**Pull request merged**